lib/declare_schema/dsl.rb

```diff
@@ -17,9 +17,9 @@ def initialize(model, **options)
 
     attr_reader :model
 
-    def timestamps
-      field(:created_at, :datetime, null: true)
-      field(:updated_at, :datetime, null: true)
+    def timestamps(**options)
+      field(:created_at, :datetime, null: false, **options)
+      field(:updated_at, :datetime, null: false, **options)
     end
```

Comment:
Is this a breaking change? Will this cause all `created_at` and `updated_at` fields to need migrations?

Reply:
🤔 It could be considered breaking, but there is the ability to have previous functionality by using `timestamps null: true` to override the new default. I think that's enough to not require the breaking change bump, but not strongly opposed.

Reply:
>It could be considered breaking but there is the ability to have previous functionality by using timestamps null: true to override the new default

This is definitely breaking since it requires code changes, its just good practice to provide an easy way of going back to old behavior 😅. This should also be a minor version at the minimum since this is adding new functionality too.

I don't know if other people are really using this gem besides invoca but if there are, we should definitely make this a breaking change.

Or we can split this into two PRs:
1. Minor version bump: Add new functionality of passing options to timestamp fields.
2. Major version bump: Change default value of `null: true` to `null: false`

Reply:
If you don't think there are other users of our gem, I'm fine with doing this as a minor version bump

Reply:
That's a good point. [Dependents graph](https://github.com/Invoca/declare_schema/network/dependents) says only 4 other users, so minor seems good for this

Reply:
But, I agree we should follow best practice here. I'll cut it as major. fd8668ea578748cf02fb002ae8d5885acae4d762